_pages/author-aswin-gopinathan.html

```diff
@@ -0,0 +1,26 @@
+---
+title: "Aswin Gopinathan"
+layout: default
+permalink: "/author-aswin-gopinathan.html"
+---
+<div class="container">
+<div class="row justify-content-center">
+    <div class="col-md-8">
+        <div class="row align-items-center mb-5">
+            <div class="col-md-9">
+                <h2 class="font-weight-bold">{{page.title}} <span class="small btn btn-outline-success btn-sm btn-round"><a href="{{ site.authors.prakruthi.twitter }}">Follow</a></span></h2>
```

Comment:
Hi @sivolko, can you fix the twitter link for this one? Cross check with other pages too.

Reply:
oh, Thanks @srishti-nema  for review ! Now I have replaced twitter link  for every pages with proper one, but some Jekyll site CI / build (pull_request) is getting failed . Kindly check it.